cdc/owner.go

```diff
@@ -85,6 +85,15 @@ func (o *Owner) getMinGCSafePointCache(ctx context.Context) model.Ts {
 			return o.minGCSafePointCache.ts
 		}
 		o.minGCSafePointCache.ts = oracle.ComposeTS(physicalTs-(o.gcTTL*1000), logicalTs)
+		// get minimum safepoint across all services
+		actualGCSafePoint, err := o.pdClient.UpdateServiceGCSafePoint(ctx, CDCMinGCSafePointCacheServiceID, CDCMinGCSafePointCacheServiceTTL, o.minGCSafePointCache.ts)
+		if err != nil {
+			log.Warn("Fail to get actualGCSafePoint from PD.", zap.Error(err))
```

Comment:
Do we need handling the error? Note that `actualGCSafePoint` would be `0` if `err != nil`.